packages/firebase_crashlytics/firebase_crashlytics_platform_interface/test/platform_interface_tests/platform_interface_crashlytics_test.dart

```diff
@@ -139,7 +128,8 @@ void main() {
 
     test('throws if .setCrashlyticsCollectionEnabled', () {
       try {
-        firebaseCrashlyticsPlatform.setCrashlyticsCollectionEnabled(true);
+        firebaseCrashlyticsPlatform!.setCrashlyticsCollectionEnabled(true);
+        // ignore: avoid_catching_errors, acceptable as UnimplementedError usage is correct
```

Comment:
If you're motivated, these could all be refactored to use `expect(() => ..., throwsUnimplementedError)`

And if you want to go fancy, we can also do:

```dart
expect(
  () => ...,
  throwsA(
    isA<UnimplementedErorr>()
      .having((e) => e.message, 'message', 'setCrashlyticsCollectionEnabled() is not implemented')
  ),
)
```

Reply:
+1 for this - can we do it on a separate PR though after this has been merged? 🤓

Reply:
I'll also add this to the backlog.

Reply:
Tracking here INVERTASE/FF-20